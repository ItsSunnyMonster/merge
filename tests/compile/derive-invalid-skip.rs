use merge::Merge;

#[derive(Merge)]
struct S {
    #[merge(skip = true)]
    field1: u8,
}

fn main() {}
