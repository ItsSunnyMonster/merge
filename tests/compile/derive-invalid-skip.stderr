error: expected `,`
 --> tests/compile/derive-invalid-skip.rs:5:18
  |
5 |     #[merge(skip = true)]
  |                  ^
